tests/baselines/reference/isolatedDeclarationsAllowJs.errors.txt

```diff
@@ -0,0 +1,12 @@
+error TS5053: Option 'allowJs' cannot be specified with option 'isolatedDeclarations'.

```

Comment:
😔We'll definitely have to make a work item to remove this restriction. Other than type nodes needing to be looked up in JSDoc comments (and having some looser expando function inference rules), JS isn't particularly special, nor should it be. TS is, after all, mostly a superset.

Reply:
I think we ca do the work as well. It's just that we have not tested this, and have not looked at this enough to know it works. Adding this now is risky.